src/View/User/show.html.twig

```diff
@@ -1,84 +1,86 @@
 {% extends 'layout.html.twig' %}
 
 {% block stylesheet %}
-  {{ parent() }}
-  <link rel="stylesheet" href="/assets/css/show.css">
-  <link rel="stylesheet" href="/assets/css/userForm.css">
+	{{ parent() }}
+	<link rel="stylesheet" href="/assets/css/show.css">
+	<link rel="stylesheet" href="/assets/css/userForm.css">
 {% endblock %}
 
 {% block title %}
-Personal Profile - {{ parent() }}
+	Personal Profile -
+	{{ parent() }}
 {% endblock %}
 
 {% block content %}
 
-<aside id="id01" class="modal">
-<div class="modal-content">
-<section class="formSection">
-  <span onclick="document.getElementById('id01').style.display='none'" class="close" title="Close Modal">&times;</span>
-  {# <section class="formSection formSectionModale"> #}
+	<aside id="id01" class="modal">
+		<div class="modal-content">
+			<section class="formSection">
+				<span onclick="document.getElementById('id01').style.display='none'" class="close" title="Close Modal">&times;</span>
+				{# <section class="formSection formSectionModale"> #}
 
     <h3 class="deleteWarning">Are you sure you want to delete your account {{user.nickName}}?</h3>
 
-    <div class="modalButtons">
-      <button type="button" onclick="document.getElementById('id01').style.display='none'" class="cancelButton">Cancel</button><br>
-      <button onclick="location.href='delete?id={{ user.id }}'" type="button" class="redButton">Delete account</button>
-    </div>
-    </section>
-    </div>
-    
-   
-</aside>
+				<div class="modalButtons">
+					<button type="button" onclick="document.getElementById('id01').style.display='none'" class="cancelButton">Cancel</button><br>
+					<button onclick="location.href='delete?id={{ user.id }}'" type="button" class="redButton">Delete account</button>
+				</div>
+			</section>
+		</div>
 
```

Comment:
please wrap all the content of the page (modal unincluded) inside a main tag
Where is the H1 ? change hierarchy from h2 h3 to h1 h2